### PR TITLE
[docs] add `case-police`, fix warnings

### PR DESCRIPTION
**Pull request by Simek**
# Why

Correct case of the software or technologies not always is that straight forward, to help maintain this automatically the `case-police` package has been created.

# How

This PR adds `case-police` to the `docs` project as `lint-case` command and fixes all reported warnings. This tool is similar to ESLint, and also offers `--fix` flag, which streamlines the process of correction even more.

The` abbreviations` preset has been disabled, because it produces too many false-positives.

Command has also been added to the docs CI workflow.

# Test Plan

Running `yarn lint-case` in `docs` do not output any warnings.

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/versions/v40.0.0/sdk/notifications.md [hunk 1]
```diff
@@ -953,7 +953,7 @@ The schedulable notification trigger you would like to check next trigger date f
 
 #### Returns
 
-If the return value is `null`, the notification won't be triggered. Otherwise, the return value is the Unix timestamp in milliseconds at which the notification will be triggered.
+If the return value is `null`, the notification won't be triggered. Otherwise, the return value is the UNIX timestamp in milliseconds at which the notification will be triggered.
```
Comment (brentvatne) [inline docs/pages/versions/v40.0.0/sdk/notifications.md:956]: meh

**Review by brentvatne — APPROVED** [review]
thanks! great idea to automate this

File: docs/pages/versions/v40.0.0/sdk/notifications.md (hunk 1 above)
Comment (brentvatne) [reply docs/pages/versions/v40.0.0/sdk/notifications.md:956]: let's change UNIX to Unix in that case

File: docs/expo-dict.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "android emulator": "Android Emulator",
+  "apple developer": "Apple Developer",
+  "eas": "EAS",
+  "expo go": "Expo Go",
+  "expo sdk": "Expo SDK",
+  "ios simulator": "iOS Simulator",
+  "react navigation": "React Navigation",
+  "unix": "Unix"
+}
```
Comment (tsapeta) [inline docs/expo-dict.json:10]:
```suggestion
}

```

**Comment by Simek** [comment]
@brentvatne I have added the custom Expo dict with overwrite rule for Unix as well as few other words and phrases:
* https://github.com/expo/expo/blob/02bd3ab9f777c2a766e2f24ea563c1e2575430a6/docs/expo-dict.json

**Review by tsapeta — APPROVED** [review]

**Review by brentvatne — APPROVED** [review]

**Pull request merged**